### PR TITLE
Lyrics/genius command

### PR DESCRIPTION
**Pull request by perronosaurio**
> New command!

### CODE REVIEW

**Comment by pedrofracassi** [comment]
> perro banned?
why not

**Comment by perronosaurio** [comment]
**i need help with the package.json xd**

**Comment by Doges** [comment]
lemme help you

File: src/commands/music/lyrics.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  async run ({ t, author, channel }, song) {
+    channel.startTyping()
+    const embed = new SwitchbladeEmbed(author)
+    const request = await lyrics.getLyrics(`search?q=${song}`)
+    const data = request.response.hits[0]
+    if (!data) {
+      embed.setColor(Constants.ERROR_COLOR).setTitle(t('commands:lyrics.noLyricsFound'))
+      return channel.send(embed).then(() => channel.stopTyping())
```
Comment (Doges) [inline src/commands/music/lyrics.js]: that return 😩

**Comment by Doges** [comment]
very outdated repo, closing PR.

**Comment by davipatury** [comment]
what's the problem? just merge the dev branch and fix the conflicts

**Comment by Doges** [comment]
@davipatury Pedro already closed another pr due it was outdated

**Comment by davipatury** [comment]
But the other PR was changing the outdated stuff.

**Comment by Doges** [comment]
btw, there's missing things in there

**Comment by perronosaurio** [comment]
Sorry i'm retarded, I'll go fix everything

File: src/utils/LyricsUtils.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const snekfetch = require('snekfetch')
```
Comment (pedrofracassi) [inline src/utils/LyricsUtils.js]: This should actually be inside of the APIs folder. APIs only load if they have the required env variables, and the commands that need them only load if they've loaded.

**Comment by pedrofracassi** [comment]
TF?

**Pull request closed without merging**